### PR TITLE
NxScrollable RSC-96 

### PR DESCRIPTION
**Pull request by glenhunter**
https://issues.sonatype.org/browse/RSC-96

Example page and basic documentation

### CODE REVIEW

File: gallery/src/styles/NxScrollable/NxScrollablePage.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      <p>
+        <code className="nx-code">.nx-scrollable</code> is a class that turns any block level element into a scrolling
+        container. It has a default max-height of 400px so in most instances you're going to want to create a modifier
+        class (e.g. <code className="nx-code">.nx-scrollable--my-box</code>) to adjust the height to suit your needs.
+      </p>
+      <p>
+        Generally <code className="nx-code">.nx-scrollable</code> is applied to blocks like
+        <code className="nx-code">.nx-tile-content</code> or <code className="nx-code">.nx-modal-content</code>.
+        <code className="nx-code">.nx-scrollable</code> should <strong>not</strong> be used to replicate page or
+        browser scrolling, instead refer to the <a href="#Pages/Page Layout/">page layout templates</a>.
```
Comment (Blacktiger) [inline gallery/src/styles/NxScrollable/NxScrollablePage.tsx:27]:
```suggestion
        browser scrolling, but instead it should refer to the <a href="#Pages/Page Layout/">page layout templates</a>.
```

or maybe

```suggestion
        browser scrolling. Instead it should refer to the <a href="#Pages/Page Layout/">page layout templates</a>.
```

File: gallery/src/styles/NxScrollable/NxScrollablePage.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+import { GalleryDescriptionTile, GalleryExampleTile } from '../../gallery-components/GalleryTiles';
+
+import NxScrollableExample from './NxScrollableExample';
+
+const nxScrollableExampleCode = require('!!raw-loader!./NxScrollableExample').default;
+
+const NxScrollablePage = () =>
+  <>
+    <GalleryDescriptionTile>
+      <p>
```
Comment (rpokorny) [inline gallery/src/styles/NxScrollable/NxScrollablePage.tsx:18]: Don't forget `.nx-p` everywhere.

File: gallery/src/styles/NxScrollable/NxScrollablePage.tsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        <code className="nx-code">.nx-scrollable</code> is a class that turns any block level element into a scrolling
+        container. It has a default max-height of 400px so in most instances you're going to want to create a modifier
+        class (e.g. <code className="nx-code">.nx-scrollable--my-box</code>) to adjust the height to suit your needs.
+      </p>
+      <p>
+        Generally <code className="nx-code">.nx-scrollable</code> is applied to blocks like
+        <code className="nx-code">.nx-tile-content</code> or <code className="nx-code">.nx-modal-content</code>.
+        <code className="nx-code">.nx-scrollable</code> should <strong>not</strong> be used to replicate page or
```
Comment (rpokorny) [inline gallery/src/styles/NxScrollable/NxScrollablePage.tsx:26]: There are several instances of missing whitespace through this section: 
image

File: gallery/src/styles/NxScrollable/NxScrollablePage.tsx (hunk 1 above)
Comment (rpokorny) [reply gallery/src/styles/NxScrollable/NxScrollablePage.tsx:27]:
```suggestion
        browser scrolling. For guidance on implementing page-level scrolling, refer to the <a href="#Pages/Page Layout/">page layout templates</a>.
```

File: gallery/src/styles/NxScrollable/NxScrollablePage.tsx (hunk 3 above)
Comment (glenhunter) [reply gallery/src/styles/NxScrollable/NxScrollablePage.tsx]: > There are several instances of missing whitespace through this section:
> image

https://github.com/sonatype/sonatype-react-shared-components/pull/53/commits/5a61a8602b77b72b72152ded55ebdb8de7629f9d

File: gallery/src/styles/NxScrollable/NxScrollablePage.tsx (hunk 2 above)
Comment (glenhunter) [reply gallery/src/styles/NxScrollable/NxScrollablePage.tsx]: > Don't forget `.nx-p` everywhere.

https://github.com/sonatype/sonatype-react-shared-components/pull/53/commits/da32ad01f3fe98b6246ba5b99cb8659f32c041ab

**Pull request merged**